test/rest_api/conftest.py

```diff
@@ -24,10 +24,20 @@
 
 import pytest
 import requests
+import ssl
+import sys
+
+from cassandra.auth import PlainTextAuthProvider
+from cassandra.cluster import Cluster, ConsistencyLevel, ExecutionProfile, EXEC_PROFILE_DEFAULT
+from cassandra.policies import RoundRobinPolicy
+
+sys.path.insert(1, sys.path[0] + '/../cql-pytest')
+from util import unique_name
```

Comment:
It's weird to duplicate so much code from cql-pytest/conftest.py and then use a 5-line function from util.py ;-) I wonder if we could import the full conftest.py from cql-pytest. Or maybe not all of it is relevant?
But we don't have to do this now.

Reply:
that's an interesting approach.
maybe we should have a formal scylla-pytest python module that will be easy to consume by the different tests (alternator, cql-pytest, redis, and now rest_api)